manifests/s/SWI-Prolog/SWI-Prolog/9.2.0/SWI-Prolog.SWI-Prolog.installer.yaml

```diff
@@ -5,7 +5,10 @@ PackageIdentifier: SWI-Prolog.SWI-Prolog
 PackageVersion: 9.2.0
 InstallerType: nullsoft
 Scope: machine
-UpgradeBehavior: install
+# NOTE:
+#   This package behave multiple version parallel existence as default.
+#   We should switch this property to avoid the default behaviour.

```

Comment:
```suggestion
#   This package's default behavior is to install multiple versions in parallel.
#   We should switch this property to avoid this default behaviour.
```

Rearranged the first sentence for clarity. Is this acceptable?

Reply:
I'm not English native speaker. Thanks for adjust them to improve readability. ❤️